guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
-- Payment links and QR Codes can contain an amount. When they do, your application should populate the amount field automatically
-- Indicate if an amount entered is more than the available balance
+**On-chain fees**
 
-## Setting the transaction fee
+This fee is dependent on how many other transactions are currently waiting to be processed on the base layer as a whole. The [average fee](https://ycharts.com/indicators/bitcoin_average_transaction_fee) in January 2021 was $0.63, and $28.60 in April 2021.
+
+</div>
+
+## Review & approval
 
 <div class="center" markdown="1">
+
 {% include image.html
-   image = "/assets/images/guide/payments/send/select-fee-rate.png"
-   retina = "/assets/images/guide/payments/send/select-fee-rate@2x.png"
-   alt-text = "Fee selection component with urgent, normal, and not urgent options"
-   width = 400
-   height = 470
-   caption = "G = Haitian Gourde, the local currency of Haiti."
-   layout = "float-right-desktop"
+   image = "/assets/images/guide/payments/send/confirm.png"
+   retina = "/assets/images/guide/payments/send/confirm@2x.png"
+   alt-text = "Example image"
+   caption = "A compact summary to confirm the information is accurate."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
 %}
 
-The application can automatically estimate a fee and set it for the sender. This would typically prioritize the transaction to be included in a block as soon as possible. Since the fee rate may vary if the network is busy, you can give senders more fine-grained fee controls so they can choose to optimize for faster confirmation or lower fees.
+Particularly with payments for larger amounts, it is good practice to allow users to review and confirm the information before it is sent.
 
-When allowing the sender to set fees, it is essential to communicate the estimated cost in their preferred currency and the estimated time until first confirmation. Be mindful that the way Bitcoin fees get calculated may not map to the sender's traditional financial experience. If you allow adjustment of fees, consider providing details on how it gets calculated.
+Payments initiated via invoices which don't require any additional input by the user can avoid this review step, as the whole user interaction is one of review and confirmation.
+
+Wallets that offer features for [pending limits]({{ '/guide/onboarding/protecting-a-wallet/#wallet-limits' | relative_url }})s or support [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) schemes for additional security may ask users to go through additional confirmation steps here.
 
-Human error with fee selection can lead to costly mistakes, and fee estimations are imperfect. Carefully consider if and how you expose transaction fee selection to senders.
 </div>
 
-**Variations**
-- Automatically set the fee rate
-- Allow sender to choose from presets
-- Allow sender to enter a custom fee rate (advanced)
+## Transaction processing
```

Comment:
```suggestion
## Payment processing
```

I notice we keep jumping between using the term payment and transaction. As this section is called payments I'd stick with calling them Payments.

Reply:
I use them pretty much interchangeably. Just using `Payments` can appear repetitive. 

I am confused about this anyways since reading @sbddesign's distinction that transactions are on-chain things and payments are Lightning things. We should clarify this.

Reply:
To me anyways transactions are much more complex things in bitcoin - a payment is much more simple and aligned with what an end user would be concerned with

Reply:
Asked about this in the #bitcoin-design-guide channel to get broader input.